cvat/settings/base.py

```diff
@@ -762,3 +762,13 @@ class CVAT_QUEUES(Enum):
 ABOUT_INFO = {
     "subtitle": "Open Data Annotation Platform",
 }
+
+if ONE_RUNNING_JOB_IN_QUEUE_PER_USER:
+    PERIODIC_RQ_JOBS.append(
+        {
+            "queue": CVAT_QUEUES.CLEANING.value,
+            "id": "cleanup_deferred_job_registry",
+            "func": "cvat.apps.redis_handler.cron.cleanup_deferred_job_registry",
+            "cron_string": "0 8 * * *",
```

Comment:
Do you have information how often we get such errors? Shouldn't this job be executed more often (e.g. every 1 or 2 hours) to reduce maximum waiting time for users?

Reply:
> Do you have information how often we get such errors?

I would say very rarely, so 1 run per day is a good start. In any case, we can increase the number of runs when needed.